### PR TITLE
Add strings for the validateUrl and validatePostName utils.

### LINKED ISSUE #1129: validateUrl & validatePostName still hardcoded english text
### Pre-Flight checklist

- [X] Did you check to see if this issue already exists?
- [X] This is a single feature request. (Do not put multiple feature requests in one issue)
- [X] This is not a question or discussion. (Use https://lemmy.ml/c/jerboa for that)

### Describe The Feature Request Below

create string resource for them

### PR DESCRIPTION
**Pull request by camporter**
Fixes #1129 

Pulls in Mockito. I'm not sure what the feeling about doing so is. 
Just wanted to be able to keep the tests while also mocking `Context` out.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -303,7 +303,13 @@
     <string name="sorttype_topweek">TopWeek</string>
     <string name="sorttype_topyear">TopYear</string>
     <string name="time_ago_ago">%1$s %2$s ago</string>
+    <string name="title">Title</string>
+    <string name="title_less_than_200_chars">Title cannot be > 200 chars</string>
```
Comment (twizmwazin) [inline app/src/main/res/values/strings.xml:307]: `Title cannot be more than 200 characters`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -303,7 +303,13 @@
...
+    <string name="title_min_3_chars">"Title must be at least 3 chars</string>
```
Comment (twizmwazin) [inline app/src/main/res/values/strings.xml:308]: `Title must be at least three characters` Also note the extra double quote

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -303,7 +303,13 @@
...
+    <string name="title_required">Title required</string>
     <string name="topAppBar_back">Back</string>
+    <string name="url">Url</string>
```
Comment (twizmwazin) [inline app/src/main/res/values/strings.xml:311]: `URL`, also capitalize below

**Review by twizmwazin — CHANGES_REQUESTED** [review]
Looks good besides some strings

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (camporter) [reply app/src/main/res/values/strings.xml]: Updated

**Review by MV-GH — APPROVED** [review]

**Review by twizmwazin — APPROVED** [review]

**Pull request merged**